src/app/public/modules/sky-tree-view/sky-tree-view-toolbar.component.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  TreeModel,
+  TreeNode
+} from 'angular-tree-component';
+
+import {
+  SkyTreeViewOptions
+} from './types/sky-tree-view-options';
+
+@Component({
+  selector: 'sky-tree-view-toolbar',
+  templateUrl: './sky-tree-view-toolbar.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyTreeViewToolbarComponent {
+
+  @Input()
+  public treeModel: TreeModel;
+
+  @Input()
+  public options: SkyTreeViewOptions;
```

Comment:
Are these options specific to the toolbar? If so, the name of the type should probably reflect that, `SkyTreeViewToolbarOptions`.

Reply:
Actually, are these even used anywhere?

Reply:
Yes, these are the additional options that were needed to cover gaps in our requirements. Today, that's just `leafNodeSelectionOnly`, but it could grow into more. They are used in several parts of the component.